CONTRIBUTING.md

```diff
@@ -29,7 +29,10 @@ See `git help commit`:
 2. Remember to sign off your commits as described above
 3. Submit a pull request
 
-***NOTE***: In order to make testing and merging of PRs easier, please submit changes to multiple charts in separate PRs.
+***NOTES***: 
+
+* In order to make testing and merging of PRs easier, please submit changes to multiple charts in separate PRs.
+* In order to make changes to the fluent-operator Helm chart, please submit changes to the [fluent/fluent-operator Helm chart](https://github.com/fluent/fluent-operator/tree/master/charts/fluent-operator). The chart in this repository will be synced synced to the release chart in fluent/fluent-operator whenever there is a new release for fluent-operator.
```

Comment:
```suggestion
* In order to make changes to the fluent-operator Helm chart, please submit changes to the [fluent/fluent-operator Helm chart](https://github.com/fluent/fluent-operator/tree/master/charts/fluent-operator). The chart in this repository will be synced to the release chart in fluent/fluent-operator whenever there is a new release for fluent-operator.
```

Reply:
Good spot and thank you for your review - apologies for lack of attention to detail. Removed the duplicate word in the latest commit.